### DISCUSSION TITLE
Select an option from js side

### DISCUSSION THREAD

**Discussion by amhuraib** [sa-si-dev/virtual-select#76] [Q&A] [answered]
Hi,

Is there a way, property or method to select an option from javascript or JQuery code?

**Comment by sa-si-dev** [comment, marked as answer]
To select option(s) on initalize use `selectedValue` property. ([more details](https://sa-si-dev.github.io/virtual-select/#/examples?id=preselect-value))

To select option(s) after initalized use `setValue()` method. ([more details](https://sa-si-dev.github.io/virtual-select/#/methods?id=setvalue))
